routersploit/modules/libs/lzs.py

```diff
@@ -0,0 +1,140 @@
+# !/usr/bin/env python
```

Comment:
Yep I saw it when looking at the Travis build. This is why I created "issue" #144... It's not viable nor a good practice to put all these (future) libs into a single file... 
Shouldn't we take time and find a better solution now ? :) I can do the job, but I dunno how you would implement it.